libfuse/debug.go

```diff
@@ -0,0 +1,34 @@
+// Copyright 2016 Keybase Inc. All rights reserved.
+// Use of this source code is governed by a BSD
+// license that can be found in the LICENSE file.
+
+package libfuse
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/keybase/client/go/logger"
+)
+
+// MakeFuseDebugFn returns a function that logs its argument to the
+// given log, suitable to assign to fuse.Debug.
+func MakeFuseDebugFn(
+	log logger.Logger, superVerbose bool) func(msg interface{}) {
+	return func(msg interface{}) {
+		str := fmt.Sprintf("%s", msg)
+		// If superVerbose is not set, filter out Statfs and
+		// Access messages, since they're spammy on OS X.
+		//
+		// Ideally, bazil would let us filter this better, and
+		// also pass in the ctx.
+		if !superVerbose &&
+			(strings.HasPrefix(str, "<- ") ||
```

Comment:
What do you think about using a single `regexp` call rather than four separate `strings` calls?  Specifically the two `Contains` calls will mean each non-matching log line is scanned twice, instead of once, which seems a little wasteful.